### PR TITLE
Implement UI tx index action rows for messages

### LINKED ISSUE #953: Implement UI tx index action rows for messages in #949

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #953

### CODE REVIEW

File: scan/src/rpc/TxHook.re [hunk 1]
```diff
@@ -409,17 +501,22 @@ let atHash = txHash => {
 };
 
 let atHeight = (height, ~page=1, ~limit=25, ()) => {
-  let json = AxiosHooks.use({j|txs?tx.height=$height&page=$page&limit=$limit|j});
+  let json =
+    AxiosHooks.use({j|txs?tx.height=$height&page=$page&limit=$limit|j});
```
Comment (sorawit) [inline scan/src/rpc/TxHook.re]: Please set line length to 100 so your autoformat won't conflict with others. Ask @Benzbeeb how to do that. He did it a few weeks ago.

**Review by sorawit — CHANGES_REQUESTED** [review]

File: scan/src/components/TxIndexPageTable.re [hunk 2]
```diff
@@ -428,6 +434,375 @@ let renderEditOracleScript = (msg, oracleScript: TxHook.Msg.EditOracleScript.t)
...
+             weight=Text.Semibold
+             code=true
+           />
+         </div>
+       </div>
+     </Col>
+     <Col> <HSpacing size=Spacing.md /> </Col>
+   </Row>;
+ };
+ */
```
Comment (evilpeach) [inline scan/src/components/TxIndexPageTable.re]: remove?

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — DISMISSED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by taobun — DISMISSED** [review]

File: scan/src/reusable/PubKeyRender.re [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+let make = (~pubKey, ~position=Text) => {
+  let noPrefixAddress = pubKey |> PubKey.toBech32 |> Js.String.sliceToEnd(~from=14);
+
+  <div className=Styles.container>
+    <Text
+      value="bandvalconspub"
+      size={position |> prefixFontSize}
+      weight=Text.Semibold
+      code=true
+      spacing={position |> letterSpacing}
```
Comment (evilpeach) [inline scan/src/reusable/PubKeyRender.re:53]: add `height={Text.Px(16)}` to make it align with the right text.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by sorawit — APPROVED** [review]
Looks good. Let's ship it!

**Pull request merged**